### ISSUE TITLE
Recreate Buildicus Concierge page into EditLLC web shop page

### ISSUE THREAD

**Issue by rodneystrong** [EditLLC/edit-web-shop#2]
You have near-complete control over this. Your stakeholder is me, Richard second, Irma a distant third.

Stick with the same design aesthetic as the site (editllc.com) but feel free to test boundaries with us. 

Add or remove content that's currently on the Buildicus Concierge page as you see fit. 

I imagine you'll consult heavily with me during the first few days of this and then gradually become independent. 

due:
12-20-15  12p

**Comment by rodneystrong** [comment]
Also, pull help from @vanderbeam and @rampage1213 as necessary.

**Comment by irms** [comment]
* ahem * distant? Hahahahaha.

**Comment by rodneystrong** [comment]
@jperez4509 , this project has shifted in terms of scope and delegation. It was decided that Jona would be taking over the design of both this and 2 of our other company sites, partly to maintain consistency. 

Jona might pull you and the other interns for help though. 

Regardless, please snap photos of your sketches so far, post them here and in our google drive, and give a brief explanation of your choices.

After that, close this issue.

due:
12-20-15  1p

**Comment by jperez4509** [comment]
![img_7843](https://cloud.githubusercontent.com/assets/8659273/11754750/35dd0c8e-a002-11e5-8525-ddb566d81bd0.JPG)

This is my very brief wire frame rough draft. This is the main home page for Edit Web Shop, the section consist of a background video, about section, services, web development, logo design, and a portfolio.

Background Video:
When the visitor first visits the site they will be greeted with the company tagline and a background video on a loop. The call to action is a play button that will play the video that was created for Edit Web Shop, after the video has completed playing a second call to action would appear that will allow the visitor the opportunity to contact us to get more info.

About Us:
This is where the paragraph that is also being used on the flyer will be placed. This will inform the visitor of what we do and how we do it. Underneath of the paragraph there would also be a testimonial (testimonials will be sprinkled through out the site.)

Services:
This section will have an icon with a brief description of the main services we provide. This section would eventually have a link to a separate for each service with more information on it.

Web Development:
This would have a description of what we do for websites and have an illustration on the left with bullets pointing to images explaining how we make all sites responsive.

Logo Design:
The section will educate the visitor on the options they have on logo design: icon, wordmark, and combination. It will also have an illustration on past logo's and how we created them.

Portfolio:
The section is very brief and works more as a cta than actually showcase all of our work. The cta section will have a parallax affect and will have a button that links to a different page with the top 4 sites and the case studies. Underneath this section there will be a row slider similar to the one I did on Paparrazme. This will have images of the sites we have created and the image will be a link to the live site.

------------------

The above bullets are just initial ideas that Jona can keep, edit or delete.
